index.js

```diff
@@ -0,0 +1,242 @@
+//import libraries
+const dotenv = require("dotenv").config();
+const phoenix = require("@phoenixlan/phoenix.js");
+const amqp = require("amqplib/callback_api");
+const {
+    UTCDate
+} = require("@date-fns/utc");
+const {
+    addHours,
+    subMonths,
+    differenceInMilliseconds,
+    differenceInHours
+} = require("date-fns");
+const {
+    Client,
+    GatewayIntentBits,
+    EmbedBuilder,
+    Guild
+} = require("discord.js");
+//initialise the api
+phoenix.init(process.env.INIT_URL);
+//change this to token from login in the api
+phoenix.User.Oauth.setAuthState(process.env.TOKEN, process.env.REFRESH_TOKEN);
```

Comment:
This is a bit unstable. We should properly authenticate the bot.

Reply:
will be fixed in new issue